### DISCUSSION TITLE
What is the "Seat Usage" means?

### DISCUSSION THREAD

**Discussion by naive231** [community/community#72223] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I saw "Copilot Monthly Usage" from the backend of my enterprise of Github:

![telegram-cloud-document-5-6181586939817233207](https://github.com/community/community/assets/650107/23a73c0b-ec41-4b95-9009-dea6ec888ae5)

I noticed that the "Seat Usage" numbers are floating number. Do anyone knows what is its definition?

**Comment by naive231** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I saw "Copilot Monthly Usage" from the backend of my enterprise of Github:
> 
> ![telegram-cloud-document-5-6181586939817233207](https://user-images.githubusercontent.com/650107/277930555-23a73c0b-ec41-4b95-9009-dea6ec888ae5.jpg)
> 
> I noticed that the "Seat Usage" numbers are floating number. Do anyone knows what is its definition?

I found the description from documentations: https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage#about-your-github-copilot-usage

It seems that if we're keeping using Copilot, the Seat usage will reach to the licenses amount we bought. But we don't know what is the definition of "using"? Is that mean the "active state" of each seat?

**Comment by queenofcorgis** [comment, marked as answer]
A Copilot seat is a license to use Copilot, which is granted to a unique user account through an organization's Copilot Business plan. Each month, the organization is charged for the number of assigned seats.

For more information, please check the official GitHub Documentation: [About billing for GitHub Copilot in your organization](https://docs.github.com/en/enterprise-cloud@latest/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-the-copilot-plan-for-your-organization/about-billing-for-github-copilot-in-your-organization)
